ddoser.py

```diff
@@ -44,7 +47,11 @@ async def make_request(url: str, proxy: Proxy, timeout: int, headers: Dict[str,
         client_session.headers.update(headers)
 
         async with client_session as session:
-            async with session.get(url, ssl=False, **request_kwargs) as response:
+            cookies = DDOS_GUARD_COOKIE_CACHE.get(base_url)
+            async with session.get(url, cookies=cookies, ssl=False, **request_kwargs) as response:
+                if response.status == HTTPStatus.FORBIDDEN and base_url not in DDOS_GUARD_COOKIE_CACHE:
+                    ddos_guard_cookies = await ddos_guard.bypass(url, response.cookies, session=session)
```

Comment:
That way we will call website URL twice:
1. to check for 403
2. in the ddos_guard.bypass
Need to think how to do it just once.

Reply:
I think we need to add a check for `server: ddos-guard` because not all 403 errors are ddos-guarded sites.